pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -736,7 +736,7 @@ func createEcMulInnerHinter(resolver hintReferenceResolver) (hinter.Hinter, erro
 // i.e, 1 if `x == 0`, 0 otherwise
 func newIsZeroNondetHint() hinter.Hinter {
 	return &GenericZeroHinter{
-		Name: "IsZeroConditional",
+		Name: "IsZeroNondet",
 		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
 			//> x == 0
```

Comment:
can you also update the comment here, to have both hintcode (python one and real one in json file)

Reply:
done.